### PR TITLE
Land request priority system

### PR DESCRIPTION
**Pull request by itsjackgardner**
Just a simple numeric priority system, stored in the request table so it can persist between queue statuses if something re-queues due to a failed dependency.

Admins can increment and decrement based on the importance of a PR, and requests will be bumped every time they re-queue so that merge times are decreased for PRs unfairly effected by other PRs.

### CODE REVIEW

File: src/routes/api/index.ts [hunk 1]
```diff
@@ -303,19 +313,24 @@ export function apiRoutes(runner: Runner, client: BitbucketClient, config: Confi
...
+        return res.status(400).json({ err: 'req.body.priority expected', body: req.body });
+      }
+      const { priority } = req.body;
+      Logger.verbose('Updating priority of land request', {
+        namespace: 'routes:api:priority',
         requestId,
+        priority,
       });
-      const success = await runner.moveRequestToTopOfQueue(requestId, req.user!);
+      const success = await runner.updateLandRequestPriority(requestId, priority);
```
Comment (Blasz) [inline src/routes/api/index.ts:329]: What happens if priority is an invalid type, does it just 500?

**Review by Blasz — APPROVED** [review]
Nice one, LGTM. Is there a staging link where we can see this in action?

File: src/routes/api/index.ts (hunk 1 above)
Comment (itsjackgardner) [reply src/routes/api/index.ts:329]: Hmm, it will 500 but would probably be better if it responded with 400 instead.

**Comment by itsjackgardner** [comment]
> Nice one, LGTM. Is there a staging link where we can see this in action?

It's live here: https://atlassian-frontend-landkid.dev.services.atlassian.com/current-state/

**Pull request merged**